hazelcast-jet-core/src/test/java/com/hazelcast/jet/impl/connector/JmsSourceIntegrationTest_RabbitMQ.java

```diff
@@ -41,4 +41,9 @@
     protected SupplierEx<ConnectionFactory> getConnectionFactory() {
         return FACTORY_SUPPLIER;
     }
+
+    @Override
```

Comment:
Does it make sense to add `@Ignore` annotation as well so it is reported as ignored?

Reply:
The `@Ignore` annotation is ignored, I tried it :)